src/pages/Edit/MessageEditPage.jsx

```diff
@@ -35,18 +34,42 @@ const MessageEditPage = () => {
   const handleInputChange = (e) => {
     setMsgText(e.target.value);
   };
+  const handleRadioChange = (e) => {
+    setName(e.target.value);
+  };
+  const [name, setName] = useState('nickname');
+
   return (
     <SLayout>
       <BackHeaderComponent text='축하메세지' />
```

Comment:
```suggestion
      <BackHeaderComponent />
```
여기 디자인 변경 있는 거 같은데 확인해주세요~

Reply:
오,,,,수정하는걸 까먹었네요ㅜㅜ고쳤습니다!